src/awsexplorer/moreResultsNode.ts

```diff
@@ -14,17 +14,17 @@ import { inspect } from 'util'
  * Clicking the node executes the Load More command for the parent Node.
  */
 export class MoreResultsNode extends AWSTreeNodeBase {
-    public constructor(public parent: LoadMoreNode) {
+    public constructor(public getParent: () => LoadMoreNode) {
```

Comment:
Might be worth a comment in this case to describe why it's a function since it looks very out of place.

Fun fact: another way to prevent certain properties from being serialized is through `Object.defineProperty`. For example, this could be placed in the constructor:
```js
Object.defineProperty(this, 'parent', { value: parent })
```
But doing this isn't 'type-safe' since we've just overwritten the constructor initializer. The above code would create a read-only 'locked' property which is not expressed through types.

Reply:
Comment added.

Reply:
This implementation was removed because of breaking errors when trying to loadMoreChildren.